projects/packages/jetpack-mu-wpcom/src/features/wpcom-comments/wpcom-comment-like.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * External dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+
+const displayNotice = message => {
+	const headerEnd = document.querySelector( '.wp-header-end' );
+	if ( ! headerEnd ) {
+		return;
+	}
+
+	// Create notice container.
+	const notice = document.createElement( 'div' );
+	notice.className = 'notice notice-error is-dismissible';
+	notice.innerHTML = `<p>${ message }</p>`;
```

Comment:
I assume message isn't escaped anywhere? If not it should be escaped here, just in case translation brings something in.

Reply:
I [suggested](https://github.com/Automattic/jetpack/pull/41475#discussion_r1965794660) letting the server generate the whole notice, but if we keep it client-side then let's **definitely** create the P node programatically and use `.innerText`.

Reply:
Update the implementation. Now all nodes are created programatically and switched from `innerHtml`to `innerText`